test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -293,5 +293,23 @@ public void CanGetElementSubComponenets()
             CollectionAssert.AreEqual(expectedRailingSubComponents, resultRailingSubComponents);
 
         }
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanGetElementTypeByName()
+        {
+            // Arrange - get element from model
+            string elementTypeName = "24\" x 24\"";
+            string emptyTypeName = "";
```

Comment:
also add a test case for searching for a valid type name that can't be found , which would throw `NotFoundException`